resources/dicts/events/ceremonies/ceremony-master.json

```diff
@@ -2269,7 +2269,7 @@
       "grumpy",
       "gloomy"
     ],
-    "(old_name) remains completely quiet as l_n names {PRONOUN/m_c/object} m_c after {PRONOUN/m_c/poss} r_h. Before anyone can congratulate {PRONOUN/m_c/object}, however, {PRONOUN/m_c/subject} {VERB/m_c/turn/turns} and walks away without a word to do {PRONOUN/m_c/poss} silent vigil."
+    "(old_name) remains completely quiet as l_n names {PRONOUN/m_c/object} m_c after {PRONOUN/m_c/poss} r_h. Before anyone can congratulate {PRONOUN/m_c/object}, however, {PRONOUN/m_c/subject} {VERB/m_c/turn/turns} and walk away without a word to do {PRONOUN/m_c/poss} silent vigil."
```

Comment:
I think "walk" should actually be verb tagged as well

Reply:
done!